mobify/platform-scaffold
# WEB-1543: Add microdata for Breadcrumbs to PDP and PLP

This PR adds microdata to breadcrumbs on the PDP and PLP. 

 **JIRA**: https://mobify.atlassian.net/browse/WEB-1543
 **Linked PRs**: https://github.com/mobify/progressive-web-sdk/pull/860

## Changes
- Added includeMicroData bool to breadcrumbs on the PDP, PLP and search results page

## Todos

## How to test-drive this PR
- cd into the SDK branch WEB-1543__breadcrumb-seo-data
- run `npm run prod:build`
- run `npm link`
- cd back to the scaffold
- run `npm link progressive-web-sdk`
- run `npm start`
- Preview merlins potions
- navigate to a PDP
- copy the markup for the PDP (starting at the '.t-product-details' container)
- paste the markup into the structured data testing tool: https://search.google.com/structured-data/testing-tool
- the data should validate and contain information on the pages breadcrumbs
- repeat using a PLP
- repeat the steps using the SFCC connector and SFCC demo site
